ee/clickhouse/sql/events.py

```diff
@@ -0,0 +1,108 @@
+DROP_EVENTS_TABLE_SQL = """
+DROP TABLE events
+"""
+
+DROP_EVENTS_WITH_ARRAY_PROPS_TABLE_SQL = """
+DROP TABLE events_with_array_props_view
+"""
+
+DROP_MAT_EVENTS_WITH_ARRAY_PROPS_TABLE_SQL = """
+DROP TABLE events_with_array_props_mv
+"""
+
+DROP_MAT_EVENTS_PROP_TABLE_SQL = """
+DROP TABLE events_properties_view
+"""
+
+EVENTS_TABLE_SQL = """
+CREATE TABLE events
+(
+    id UUID,
+    event VARCHAR,
+    properties VARCHAR,
+    timestamp DateTime64(6, 'UTC'),
+    team_id Int32,
+    distinct_id VARCHAR,
+    elements_hash VARCHAR,
+    created_at DateTime
+) ENGINE = MergeTree()
+PARTITION BY toYYYYMM(timestamp)
+ORDER BY (id, timestamp, intHash32(team_id))
+SAMPLE BY intHash32(team_id)
+"""
```

Comment:
I've updated a few of the definitions on the google doc. The only one that really matters right now is this one though (events).
https://docs.google.com/document/d/1KflVt-ETRybgJnUlGFiCiWnROTbj2qNoMgOxdoC6Ir8/edit#heading=h.s8elmxezz9yp

The main changes are for replication, ordering, and the new hot -> cold storage profile.

The downside to adding this is that it will break any testing setup that you have so far. Either we will need to setup a local environment that supports ReplicatedMergeTree or we will need to have two model variants: Single node Clickhouse and Replicated Clickhouse.

Reply:
Can we just leave how it is now (single node setup) on our OSS codebase here and then replace the SQL file on posthog-production with SQL that uses replicatingmergetree? In other words, would querying change anywhere because if not we can just swap in SQL on production build